### PR TITLE
Master

### PR DESCRIPTION
**Pull request by ApacheCamel**
Had to re-download all the files from the forge website and re-run the commands. Now it should work when opening the .project file in eclipse.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# TubeCake
```
Comment (KristopherWagner) [inline README.md]: why are you removing the README?

File: TubeCake_Client.launch [hunk 2]
```diff
@@ -1,13 +1 @@
...
-</listAttribute>
-<listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
-<listEntry value="4"/>
-</listAttribute>
-<stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="GradleStart"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="TubeCake"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-DFORGE_FORCE_FRAME_RECALC=true"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="D:\gocode\src\github.com\arcite\TubeCake\run"/>
-</launchConfiguration>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication"><stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="GradleStart"/><stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="TubeCake"/><stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="C:\Users\Tyler\Documents\GitHub\TubeCake\run"/><stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value=" -DFORGE_FORCE_FRAME_RECALC=true"/></launchConfiguration>
```
Comment (KristopherWagner) [inline TubeCake_Client.launch]: This should be expanded instead of one line

File: TubeCake_Server.launch [hunk 3]
```diff
@@ -1,13 +1 @@
...
-</listAttribute>
-<listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
-<listEntry value="4"/>
-</listAttribute>
-<stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="GradleStartServer"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="TubeCake"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-DFORGE_FORCE_FRAME_RECALC=true"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="D:\gocode\src\github.com\arcite\TubeCake\run"/>
-</launchConfiguration>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication"><stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="GradleStartServer"/><stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="TubeCake"/><stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="C:\Users\Tyler\Documents\GitHub\TubeCake\run"/><stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value=" -DFORGE_FORCE_FRAME_RECALC=true"/></launchConfiguration>
```
Comment (KristopherWagner) [inline TubeCake_Server.launch]: this should be expanded instead of one line

File: build.gradle [hunk 4]
```diff
@@ -11,17 +11,17 @@ apply plugin: 'net.minecraftforge.gradle.forge'
 //Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
 
 
-version = "1.0.0"
-group = "com.crashcoursegaming.tubecake"
-archivesBaseName = "tubecake"
+version = "1.0"
+group = "com.yourname.modid" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
+archivesBaseName = "modid"
```
Comment (KristopherWagner) [inline build.gradle]: looks like you deleted our information

File: build.gradle [hunk 5]
```diff
@@ -11,17 +11,17 @@ apply plugin: 'net.minecraftforge.gradle.forge'
...
 
 sourceCompatibility = targetCompatibility = "1.6" // Need this here so eclipse task generates correctly.
 compileJava {
     sourceCompatibility = targetCompatibility = "1.6"
 }
 
 minecraft {
-    version = "1.11.2-13.20.0.2206"
+    version = "1.11-13.19.1.2199"
```
Comment (KristopherWagner) [inline build.gradle]: any reason why you switched to this version of minecraft?

File: run/config/forge.cfg [hunk 6]
```diff
@@ -1,72 +0,0 @@
-# Configuration file
```
Comment (KristopherWagner) [inline run/config/forge.cfg]: why are you deleting the minecraft run configurations?

File: run/config/forgeChunkLoading.cfg [hunk 7]
```diff
@@ -1,47 +0,0 @@
-# Configuration file
```
Comment (KristopherWagner) [inline run/config/forgeChunkLoading.cfg]: see above question

File: run/config/splash.properties [hunk 8]
```diff
@@ -1,18 +0,0 @@
-#Splash screen properties
```
Comment (KristopherWagner) [inline run/config/splash.properties]: see above question

File: setup.bat [hunk 9]
```diff
@@ -1,3 +0,0 @@
-gradlew setupDecompWorkspace
```
Comment (KristopherWagner) [inline setup.bat]: isn't this needed?

File: src/main/java/com/example/examplemod/ExampleMod.java [hunk 10]
```diff
@@ -0,0 +1,20 @@
+package com.example.examplemod;
```
Comment (KristopherWagner) [inline src/main/java/com/example/examplemod/ExampleMod.java]: don't need examples in the repo

File: src/main/resources/mcmod.info [hunk 11]
```diff
@@ -0,0 +1,16 @@
+[
+{
+  "modid": "examplemod",
```
Comment (KristopherWagner) [inline src/main/resources/mcmod.info]: is this more example mod, or are we supposed to edit it?

File: README.md (hunk 1 above)
Comment (ApacheCamel) [inline README.md]: I prefer leaving users in the dark.

**Pull request closed without merging**